compute/cker/include/cker/operation/LogicalOr.h

```diff
@@ -62,12 +62,20 @@ inline void LogicalOrBroadcast4D(const Shape &unextended_input1_shape, const T *
 }
 
 template <typename T>
-inline void LogicalOr(const Shape &unextended_input1_shape, const T *input1_data,
-                      const Shape &unextended_input2_shape, const T *input2_data,
-                      const Shape &unextended_output_shape, T *output_data)
+inline void LogicalOrElementwise(const Shape &shape, const T *input1_data, const T *input2_data,
+                                 T *output_data)
 {
-  LogicalOrBroadcast4D<T>(unextended_input1_shape, input1_data, unextended_input2_shape,
-                          input2_data, unextended_output_shape, output_data);
+  int num_elements = 1;
```

Comment:
You can use `Shape::FlatSize` function here. It returns the total count of elements.

Reply:
Thanks, got it. Make it simple.